src/data/maps.json

```diff
@@ -69,6 +69,228 @@
                             }
                         ]
                     }
+                ],
+                "labels": [
+                    {
+                        "position": [9,308],
+                        "text": "Primorsky Ave.",
+                        "rotation": -90,
+                        "size":80
+                    },
+                    {
+                        "position": [9,104],
+                        "text": "Primorsky Ave.",
+                        "rotation": -90,
+                        "size":80
+                    },
+                    {
+                        "position": [9,-80],
+                        "text": "Primorsky Ave.",
+                        "rotation": -90,
+                        "size":80
+                    },
+                    {
+                        "position": [125,10],
+                        "text": "Kilmov ST.",
+                        "size":80
+                    },
+                    {
+                        "position": [-104,28],
+                        "text": "Kilmov ST.",
+                        "rotation": 10,
+                        "size":80
+                    },
+                    {
+                        "position": [-125,210],
+                        "text": "Nikitskaya ST.",
+                        "rotation": -1,
```

Comment:
I don't think this needs rotation
```suggestion
```

Reply:
It is. If not it is of kilt from the street and my ADHD/OCD wont allow me not to.. lol

Reply:
I will up date ST to St.

Reply:
I can assure you that Nikitskaya St. and Verhnyaya St. are parallel to each other, maybe it looks like that because it gets larger in the end?

Reply:
Damn your right. I'll fix it now.

Reply:
I was using the road as a guide to line it up. But if you use the median which is straight. What a strange optical illusion.